database/src/desktop/connection/persistent_connection.cc

```diff
@@ -1080,6 +1180,55 @@ void PersistentConnection::OnAuthRevoked(Error error_code,
   realtime_->Close();
 }
 
+void PersistentConnection::RefreshAppCheckToken(const std::string& token) {
+  scheduler_->Schedule(new callback::CallbackValue2<ThisRef, std::string>(
+      safe_this_, token, [](ThisRef ref, std::string token) {
+        ThisRefLock lock(&ref);
+        if (lock.GetReference() != nullptr) {
+          auto* connection = lock.GetReference();
```

Comment:
I think `auto connection` without the `*` should be sufficient.

Reply:
Done